arch/arm/src/arm/arm_prefetchabort.c

```diff
@@ -65,16 +65,19 @@
 
 void arm_prefetchabort(uint32_t *regs)
 {
+  struct tcb_s *tcb = this_task();
+
 #ifdef CONFIG_LEGACY_PAGING
-  uint32_t *savestate;
+  bool savestate;
 
   /* Save the saved processor context in current_regs where it can be
```

Comment:
The comment here is now wrong, it talks about CPU state but no CPU state is involved here (there are many places with this comment).